packages/x-components/src/components/scroll/scroll.mixin.ts

```diff
@@ -98,6 +98,31 @@ export default class ScrollMixin extends Vue {
    * @internal
    */
   protected scrollHeight = 0;
+  /**
+   * Flag to know if the property clientHeight is changing or gets the final value.
+   *
+   * @internal
+   */
+  protected isClientHeightChanging = false;
+  /**
+   * Property for setting the direction of scroll.
+   *
+   * @internal
+   */
+  protected scrollDirection: ScrollDirection = 'UP';
+
+  /**
+   * Restores the clientHeight flag when clientHeight property stops changing.
+   * Also sets a new previous position, before update the current one.
+   *
+   * @internal
+   */
+  protected restoreClientHeightFlag(): void {
+    this.isClientHeightChanging = false;
+    this.previousPosition = this.currentPosition;
+  }
+
+  protected throtteledCall = throttle(this.restoreClientHeightFlag, 100);
```

Comment:
Can we use here `this.throttleMs` prop instead of '100' or is it not worth it?

Reply:
They are different `throttles`, so I don't know if it's correct to use the same time. Maybe in the future it could change?

Reply:
That's ok!! Thks, I wasn't sure about that!